src/lib.rs

```diff
@@ -669,6 +669,22 @@ impl LevelFilter {
     }
 }
 
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
```

Comment:
Not sure if we want equality, etc to consider staticness or not. It seems a bit weird for text to not be equal to itself depending on the lifetime, but at the same time it does affect how the Record behaves.

Reply:
Since this is publicly unreachable it seems ok to me. If we ever did want to expose some string-like type we could think about it then?